clients/gateway/gateway.go

```diff
@@ -141,6 +144,7 @@ func (c *Client) doPost(ctx context.Context, url string, data any) (*http.Respon
 	}
 
 	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("User-Agent", c.version)
```

Comment:
see how user agent should be formatted 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax